### DISCUSSION TITLE
Is there a "Getting Started" for rust?

### DISCUSSION THREAD

**Discussion by zupat** [fable-compiler/Fable#3313] [Q&A]
I want to make a small cli tool, how do I go about transpiling f# to rust?

**Comment by ncave** [comment]
@jinyus We don't have many samples yet, but you can see for example these two repos:
- https://github.com/ncave/fable-raytracer (some browser/xplat/webasm rendering).
- https://github.com/alfonsogarciacaro/fable-rust-sample (some rudimentary file io).

Note: The next upcoming Fable pre-release version will support `printfn/sprintf` for Rust, but for the current one you may have to use `Console.WriteLine`.
